lua/cfc_voice_channels/core/sv/sv_player.lua

```diff
@@ -22,9 +22,32 @@ function player:getConnectedChannel()
     end
 end
 
+function player:isInSameChannel(ply)
```

Comment:
Again, function is too vague to add to player table. We really want to be 100% sure there's never any collisions with any other addon. That would be a disaster and would be difficult to debug.

Reply:
What would you suggest? `isInSameVoiceChannel` or `isInSameCFCVoiceChannel`...?

Reply:
I like scoping it specifically to CFC - I know it's unlikely that the first one would have a collision but I' have name collision PTSD.